### ISSUE TITLE
POST /api/v1/provisioning/contact-points 500 with no context (name too long)

### ISSUE THREAD

**Issue by makarov-roman** [grafana/terraform-provider-grafana#1916]
### Terraform Version

1.9.8

### Terraform Grafana Provider Version

3.13.1

### Grafana Version

11.3.0

### Affected Resource(s)

* grafana_contact_point

### Terraform Configuration Files

```hcl
resource "grafana_contact_point" "teams_contact_point_root" {
  name   = "name-32-chars-here"
  org_id = grafana_organization.org.id

  teams {
    url                     = var.teams_webhook_url_root
    title                   = "CUSTOMER MONITORING ALERTS"
    disable_resolve_message = true
    message                 = <<EOT
{{ if gt (len .Alerts.Firing) 0 }}
  {{ len .Alerts.Firing }} alert/s firing:
  {{ range .Alerts.Firing }} {{ template "alert" .}} {{ end }}
{{ end }}
EOT
  }
}
```

### Expected Behavior

Contact point is created

### Actual Behavior

500 error with no context at all

### Important Factoids

We're were checking different scenarios which could allow to reproduce the error and turned out if you use name longer than 30 symbols via terraform provider, then you have 500, but the same name works via UI.

**Comment by meSATYA** [comment]
Also waiting for this character limitation to be removed and should be set to 64 chars long.

**Comment by meSATYA** [comment]
There is no character limitation when the contact point is created on Grafana UI. 😢 

Grafana Version: 11.3.0
Grafana Terraform Provider: 3.16.0

<img width="1791" alt="Image" src="https://github.com/user-attachments/assets/d456118d-e7d1-4db4-bdb8-c1fd1fad2025" />

**Comment by JacobsonMT** [comment]
I can't be 100% sure as there is no context in the error but I believe it likely this is the following bug: https://github.com/grafana/grafana/pull/95084

It should be fixed in Grafana v11.3.1.

**Comment by meSATYA** [comment]
Thanks @JacobsonMT. The longer character Contact Point Name issue is resolved in v11.3.1.

**Comment by makarov-roman** [comment]
It's quite strange behaviour that endpoint fails with 500 with no error message. Basically "something went wrong".
no reasons to not believe Mr @meSATYA. I will close the issue.
Thanks everyone.

**Comment by meSATYA** [comment]
Yes, the error doesn't specify what going wrong. But the very possible reason is the longer contact point name.
